Port blogs (#3)
Inferara/inferara.com: content/blog/small-and-big-step-semantics.md

+++
title = 'Small and Big Step Semantics'
date = 2024-03-21T16:16:06+13:00
draft = false
math = "katex"
tags = ["Formal Methods", "Formal Semantics"]

> summary = "This blog introduces small-step and big-step semantics, what the differences between them are, and their different applications when it comes to analyzing computer programs."

Reviewer: ```suggestion
summary = "This blog outlines small-step and big-step semantics, their differences, and applications in analyzing computer programs."
```
